MzTab_MTD

```diff
@@ -481,16 +475,17 @@ CREATE TABLE "XML"."TraML"
     "FragmentCharge" integer,
     "FragmentType" text COLLATE pg_catalog."default",
     "FragmentSeriesNumber" integer,
-    quantifying_transition boolean,
-    identifying_transition boolean,
-    detecting_transition boolean,
+    quantifying_transition integer,
+    identifying_transition integer,
+    detecting_transition integer,
     "Tr_recalibrated" double precision,
     "RetentionTime" double precision,
     "PeptideGroupLabel" text COLLATE pg_catalog."default",
     "UniprotId" text COLLATE pg_catalog."default",
     "CompoundName" text COLLATE pg_catalog."default",
     "SMILES" text COLLATE pg_catalog."default",
     "SumFormula" text COLLATE pg_catalog."default",
+    "Internal_mzTab-ID" integer NOT NULL,
```

Comment:
I am a bit confused between what is the internal ID and what is the primary key or foreign key used to join the tables.